service/main.ml

```diff
@@ -160,6 +160,15 @@ let submission_service =
     ~docv:"FILE"
     ["submission-service"]
 
+(* https://github.com/ocurrent/opam-repo-ci/issues/260 *)
+let is_macos =
```

Comment:
Why not detect this automatically? We'll otherwise have to detect it automatically in the tests, and it seems better to have that logic as part of the pipeline, even if it's only a workaround.

Reply:
This will be in a separate PR:)